### ISSUE TITLE
Mask a token in ktrain

### ISSUE THREAD

**Issue by aphuongle95** [amaiya/ktrain#183]
Hi, I just want to check, if I want to simplify my model and mask some tokens with "[MASK]", is it a correct way to do it? When I try to do predictor.explain, it prints out "mask" token without square bracket so I wonder if it treats the token as a "masked token" or a normal word "mask". Thanks in advance!

**Comment by amaiya** [comment]
Hello:  Do you mean that you're supplying "[MASK]" as input to `predictor.explain`?

**Comment by aphuongle95** [comment]
Yes, that's right.

**Comment by amaiya** [comment]
Hi @aphuongle95 : The `explain` method uses the `eli5`  (and `lime`) to generate the explanation.  It is model-agnostic and not specific to BERT, so does not understand "[MASK]" as a mask token.    Although this isn't supported right now, it may be possible for you to mask tokens yourself by invoking `eli5` yourself and using [MaskingTextSampler](https://eli5.readthedocs.io/en/latest/autodocs/lime.html#module-eli5.lime.samplers). I haven't used or tried it myself, though.
